estimator/serenity_estimator.cpp

```diff
@@ -35,14 +35,22 @@ class SerenityEstimatorProcess :
 
   Future<Resources> _oversubscribable(
       const Future<ResourceUsage>& _resourceUsage) {
-    Try<Resources> ret = this->pipeline->run(_resourceUsage.get());
+    // TODO(bplotka): Decide if need such simple algorithm.
+    Resources allocatedRevocable;
```

Comment:
I checked that issue some time ago: Resources(50) - Resources(100) = Resources(0), so we don't need to worry.